### PR TITLE
Read method for Users

### PR DESCRIPTION
**Pull request by vlasovigor**
- [x] Add method

- [x] Add test

- [x] Add exceptions
https://github.com/malinink/easy-tests/issues/326

### CODE REVIEW

**Comment by vlasovigor** [comment]
@malinink pls,look

File: src/main/java/easytests/api/v1/controllers/UsersController.java [hunk 1]
```diff
@@ -51,15 +51,26 @@ private Boolean isAdmin() {
...
-     */
+    @GetMapping("/{userId}")
+    public User show(@PathVariable Integer userId) throws NotFoundException, ForbiddenException {
+        final UserModelInterface userModel = this.usersService.find(userId);
+
+        if (userModel == null) {
+            throw new NotFoundException();
+        }
+
+        if (!this.isAdmin()) {
```
Comment (malinink) [inline src/main/java/easytests/api/v1/controllers/UsersController.java]: move admin check before user one. We should not tell to none admin users that some user id exists.

File: src/test/java/easytests/api/v1/controllers/UsersControllerTest.java [hunk 2]
```diff
@@ -128,15 +128,60 @@ public void testListForbidden() throws Exception {
...
+                                .with(email, userModel.getEmail())
+                                .with(isAdmin, userModel.getIsAdmin())
+                                .with(state, userModel.getState())
+                        .build()
+                ))
+                .andReturn();
+    }
+
+    @Test
+    public void testShowNotFound() throws Exception {
```
Comment (malinink) [inline src/test/java/easytests/api/v1/controllers/UsersControllerTest.java]: fix tests, them will fail after order check changes.

**Review by malinink — CHANGES_REQUESTED** [review]

**Comment by vlasovigor** [comment]
@malinink fixed,look again please

**Review by malinink — APPROVED** [review]

**Pull request merged**